src/components/profile/ProfileAccount.jsx

```diff
@@ -30,6 +31,72 @@ export default function ProfileAccount() {
               <div className="flex justify-end">{auth.user?.phone}</div>
             </div>
           </li>
+          <li>
+            <div className="flex justify-between">
+              <div className="flex justify-start gap-4">
+                <PlaceIcon />
+                Address
+              </div>
+              <div className="flex justify-end">
+                {auth.user?.address?.street +
+                  ", " +
+                  auth.user?.address?.city +
+                  ", " +
+                  auth.user?.address?.state +
+                  ", " +
+                  auth.user?.address?.zipcode}
+              </div>
+            </div>
+          </li>
+        </ul>
+        <ul className="menu bg-base-100 px-12 w-full rounded-box md:hidden">
+          <li>
+            <div className="flex justify-between">
+              <div className="flex justify-start gap-4">
+                <PersonIcon />
+              </div>
+              <div className="form-control w-full">
+                <label className="label">
+                  <span className="label-text">Name</span>
+                </label>
+                <div className="flex justify-start">{auth.user?.name} </div>
+              </div>
+            </div>
+          </li>
+          <li>
+            <div className="flex justify-between">
+              <div className="flex justify-start gap-4">
+                <PhoneIcon />
+              </div>
+              <div className="form-control w-full">
+                <label className="label">
+                  <span className="label-text">Phone Number</span>
+                </label>
+                <div className="flex justify-start">{auth.user?.phone} </div>
+              </div>
+            </div>
+          </li>
+          <li>
+            <div className="flex justify-between">
+              <div className="flex justify-start gap-4">
+                <PlaceIcon />
+              </div>
+              <div className="form-control w-full">
+                <label className="label">
+                  <span className="label-text">Address</span>
+                </label>
+                <div className="flex justify-start">
+                  {auth.user?.address?.street +
+                    ", " +
+                    auth.user?.address?.city +
+                    ", " +
+                    auth.user?.address?.state +
+                    ", " +
+                    auth.user?.address?.zipcode}
+                </div>
+              </div>
+            </div>
+          </li>
```

Comment:
at least this is working
I am not going to request a change.
but it's not the way to write this part.

Reply:
what do you mean this is not the way. it's what you asked before to use the md to control display content

Reply:
if you have ideas to make this better just say it